torch/_dynamo/output_graph.py

```diff
@@ -1021,6 +1028,12 @@ def call_user_compiler(self, gm: fx.GraphModule) -> CompiledFn:
             )
             unimplemented_with_warning(e, self.root_tx.f_code, msg)
         except Exception as e:
+            if (
+                e.args[0] == "Cannot convert symbols to int"
+                or isinstance(e, GuardOnDataDependentSymNode)
```

Comment:
Not sure if this covers all exceptions.

Reply:
No, just because this is prototype, I'll wrap it as a distinguished exception type.